src/Tizen.NUI/src/public/NativeImageSource.cs

```diff
@@ -22,6 +22,22 @@ internal NativeImageSource(IntPtr cPtr, bool cMemoryOwn) : base(Interop.NativeIm
             Handle = cPtr;
         }
 
+
+        /// <summary>
+        /// Get URL from native image source.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Url
```

Comment:
StyleCop(.NET Framework Design Guideline) recommends to use [Uri](https://docs.microsoft.com/en-us/dotnet/api/system.uri?view=net-5.0) Type instead of string~
Please check [CA1056](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1056) StyleCop warnings(it is the same as the warning below.)

Reply:
Thank you I fixed it